shared/js/background/url-parameters.js

```diff
@@ -28,16 +21,7 @@ function ensureTrackingParametersConfig () {
         return false
     }
 
-    plainTextTrackingParameters = []
-    regexpTrackingParameters = []
-    for (const key of
-        tdsStorage.config.features.trackingParameters.settings.parameters) {
-        if (regexpParameterTest.test(key)) {
-            regexpTrackingParameters.push(new RegExp(key))
-        } else {
-            plainTextTrackingParameters.push(key)
-        }
-    }
+    trackingParameters = tdsStorage.config.features.trackingParameters.settings.parameters
```

Comment:
Nit: Make this a Set, so it can be checked more efficiently.
```suggestion
    trackingParameters = new Set(tdsStorage.config.features.trackingParameters.settings.parameters)
```